config/crd/networking.cloudfoundry.org_routes.yaml

```diff
@@ -17,6 +17,13 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
+  additionalPrinterColumns:
```

Comment:
I think this should be done in the CRD structs. This file is one that we copy over from what Kubebuilder generates:

* https://github.com/cloudfoundry/cf-k8s-networking/blob/94e61142eab08f591dfcbc93ace54e0dae26d064/routecontroller/apis/networking/v1alpha1/route_types.go
* https://book.kubebuilder.io/reference/generating-crd.html#additional-printer-columns

Reply:
Fixed